test/integration/answer_test.rb

```diff
@@ -14,11 +14,15 @@ class AnswerTest < ActionDispatch::IntegrationTest
 
   test "related links are rendered" do
     setup_and_visit_content_item('answer')
-    within shared_component_selector("related_items") do
-      elsewhere = @content_item["details"]["external_related_links"].first
-      elsewhere_page = JSON.parse(page.text).fetch("sections").last["items"].first
-      assert_equal elsewhere["title"], elsewhere_page["title"]
-      assert_equal elsewhere["url"], elsewhere_page["url"]
+
+    related_links = @content_item["details"]["external_related_links"].first
```

Comment:
I assume you don't want to loop through all links because collapsed items complicate it, but you're just taking the first link, here, I'd say the var name should match, eg related_link or  first_related_link